src/views/virtualmachines/details/tabs/configuration/virtual-machine-configuration-tab.scss

```diff
@@ -4,6 +4,7 @@
     margin: var(--pf-global--spacer--sm);
     width: fit-content;
     min-width: max-content;
+    max-width: 100px;
```

Comment:
have u tested on various resolutions? u can using chrome dev tools

Reply:
Yes. It's working with this